pkg/server/handler/metrics/handler.go

```diff
@@ -55,11 +55,24 @@ func New(c Config) *Handler {
 
 	{
 		nam := "teams_bridge_duration_seconds"
-		cou[nam] = recorder.NewHistogram(recorder.HistogramConfig{
-			Des: "the time it takes for bridge transactions",
+		his[nam] = recorder.NewHistogram(recorder.HistogramConfig{
```

Comment:
This was the first example I created, but nobody used it so far. Just fixing what was broken.